src/reporter/coverage-report.xsl

```diff
@@ -196,19 +224,16 @@
       <!-- Analyze the entire stylesheet string. For each matching substring, create a map that
          records the kind of match. -->
       <xsl:variable name="regex-groups" as="map(xs:integer, xs:string)+">
-         <xsl:analyze-string select="$stylesheet-string"
-            regex="{$construct-regex}" flags="sx">
+         <xsl:analyze-string select="$stylesheet-string" regex="{$construct-regex}" flags="sx">
             <xsl:matching-substring>
                <xsl:map>
-                  <xsl:for-each select="1 to 10">
+                  <xsl:for-each select="1 to 11">
```

Comment:
I like how you replaced the hardcoded numbers with `$group-*` variables.
How about going further and putting the number definitions in a single variable? Then we would be able to replace this hardcoded `11` with the size of that single variable.

Reply:
Suggestion on how to do that, using a map or sequence?

Reply:
Both will do, but I guess a map would be more readable.